### PR TITLE
Split out playlist list into component + Fix bugs + add device dropdown + add sass

### PR DESCRIPTION
**Pull request by dwilliams27**
Splits out the list of playlists into separate component
Fixes duplicate text bug
Fixes multiple bugs related to playback (especially for the first song)
Creates constants file
Adds dropdown menu for devices
Fixes a lot of spacing

Adds sass, so we can make our styling more consistent

### CODE REVIEW

**Comment by dwilliams27** [comment]
Still need to work a but on the styling, things are not centered

File: src/components/Playlists.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  render () {
+    return <div>
+      <input id="playlist-filter" placeholder="Start typing to filter for your playlist" onChange={ this.filterPlaylist } value={ this.state.playlistFilter }/>
+      { this.props.playlists.length > 0 ?
+        <div>
+          <div className="playlist-container">
+            <ul>
+              <form id="playlist-select">
+              { this.props.playlists.filter((playlists) => playlists.name.toLowerCase().includes(this.state.playlistFilter.toLowerCase())).map((pl) => (
```
Comment (dwilliams27) [inline src/components/Playlists.tsx]: This line took me a bit to get right

**Comment by ColemanMitch** [comment]
> Still need to work a but on the styling, things are not centered

It's okay that the device select dropdown and the playlist filter aren't centered on the page for right now. My plan is to eventually make the interval of `Timer` configurable via a slider and have that on the other half of the page. If this doesn't make total sense I can draw a picture of it/make a mockup of what I'm envisioning.

**Comment by dwilliams27** [comment]
Added sass too (https://www.w3schools.com/sass/). Merge this whenever you think its good

**Comment by dwilliams27** [comment]
Also as a note if you ever change the scss files, you'll need to rerun `npm run build` to have the changes load in local

**Comment by dwilliams27** [comment]
oops dont merge I broke it

**Comment by dwilliams27** [comment]
Good to go

**Comment by ColemanMitch** [comment]
Merging this bad boy with main

**Pull request merged**